test/parallel_api/dynamic_selection/sycl/test_sycl_sanity.pass.cpp

```diff
@@ -0,0 +1,92 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) 2023 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <cstdio>
+#include <memory>
+#include <ostream>
+#include <vector>
+#include <CL/sycl.hpp>
```

Comment:
This include ```#include <CL/sycl.hpp>``` already exist in the ```support/sycl_sanity.h``` file so it's not required here.

Reply:
Fixed